### ISSUE TITLE
1.17

### ISSUE THREAD

**Issue by MenacingPerson** [theorbtwo/RoughlyEnoughResources#31]
REI has been updated but there's been no work on this.

**Comment by catgirldoll** [comment]
Yup. I'd like to know if a 1.17 update is still planned, or whether someone should take over the project

**Comment by MenacingPerson** [comment]
there is a button to like it, please don't fill up this issue with +1 comments.

**Comment by supersaiyansubtlety** [comment]
This should be closed thanks to 60a7e6b
